docs/docs/networks.md

```diff
@@ -18,25 +18,47 @@ Fetchhub is our mainnet v2 production network.
 
 ## Testnet v2: fetchhubtest
 
-Fetchhubtest is our testnet v2 production network. 
+Fetchhubtest is our testnet v2, running similar versions than our production network. 
 
 | Parameter      | Value                                                                                      |
 | -------------- | ------------------------------------------------------------------------------------------ |
-| Chain ID       | fetchhubtest-1                                                                             |
+| Chain ID       | fetchhubtest-2                                                                             |
 | Denomination   | atestfet                                                                                   |
 | Decimals       | 18 (1testfet = 1000000000000000000atestfet)                                                |
 | Version        | v0.7.x (fetchcli >= v0.7.2)                                                                |
-| RPC Endpoint   | https://rpc-fetchhubtest.prod.fetch-ai.com                                                 |
-| REST Endpoint  | https://rest-fetchhubtest.prod.fetch-ai.com                                                |
-| Block Explorer | [https://explore-fetchhubtest.prod.fetch-ai.com](https://explore-fetchhubtest.prod.fetch-ai.com) |
+| RPC Endpoint   | https://rpc-fetchhubtest.t-v2-london-c.fetch-ai.com:443                                    |
```

Comment:
Before we make this public it would be nice to sort out these URI's so we have `https://rpc-fetchhubtest.fetch.ai` etc.

I think that we should discuss this at the DevOps standup tomorrow

Reply:
done :)